File: group#2/김수현/1주차/1.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> lottos, vector<int> win_nums) {
    int a=0,b=0;
    for(int i=0;i<lottos.size();i++){
        if(lottos[i]!=0){
            if(find(win_nums.begin(), win_nums.end(), lottos[i])!=win_nums.end()){
                a++;
            }
        }else b++;
        
    }
    vector<int> answer;
    if(a+b==0)answer.push_back(6);
    else answer.push_back(7-a-b);
    
    if(a==0)answer.push_back(6);
    else answer.push_back(7-a);
    
    return answer;
}

File: group#2/김명기/1주차/1.py
# 로또의 최고 순위와 최저 순위
def solution(lottos, win_nums):
    num_zeros = 0
    for i in lottos:
        if i == 0:
            num_zeros +=1
    lottos = [i for i in lottos if i]
    cnt = 0
    for i in lottos:
        if i in win_nums:
            cnt+=1
    t = [6,6,5,4,3,2,1]
    return [t[cnt+num_zeros], t[cnt]]